lib/userIsAdmin.js

```diff
@@ -0,0 +1,11 @@
+export default function userIsAdmin(req) {
```

Comment:
I know it's simple, but putting this all over the place felt wrong. In the future, I could also see this expanding to have more than one level of access, for something like _ADMIN_,_CANDIDATE_,_VOTER_ if we wanted to allow candidates to edit their own information.